android/src/main/java/org/reactnative/camera/RNCameraView.java

```diff
@@ -197,9 +199,9 @@ protected void onLayout(boolean changed, int left, int top, int right, int botto
         correctHeight = (int) height;
       }
     }
-    int paddingX = (int) ((width - correctWidth) / 2);
-    int paddingY = (int) ((height - correctHeight) / 2);
-    preview.layout(paddingX, paddingY, correctWidth + paddingX, correctHeight + paddingY);
+
+    mViewFinderSize = new Size((int) width, (int) height);
+    preview.layout(0, 0, correctWidth, correctHeight);
```

Comment:
this will cause problem for other users

maybe we should provide more options to layout preview

Reply:
@sibelius thanks for the review! I'm not sure how to reliably access the `options` map from `onLayout`.  Even if I could, I would have to work out how to call `preview.layout` after changing the prop (which might change each time `takePictureAsync` is called).

I'm not sure if that would be desirable, so as far as I can tell, the options for preserving the existing behaviour are to make `cropToPreview` a component property (passed to `<RNCamera>`) rather than a property passed to `takePictureAsync`, and either
- figure out how to call `preview.layout` each time it is updated
- don't do that, and tell people in the docs that changes to `cropToPreview` won't take effect after mounting the component

I will try to do that if you like but I think it would be better to just cut a new release and mention the changes to the padding on Android in the release notes.